### DISCUSSION TITLE
Need suppport for nigerian currency

### DISCUSSION THREAD

**Discussion by HarshikaSM** [dm4t2/vue-currency-input#408] [Q&A]
Currentlty there is no support for nigerian currrency. It will be helpful if the support for the same is added, so that the value gets displayed properly.

**Comment by dm4t2** [comment]
This should already work with `{currency: 'NGN'}`, shouldn't it?

https://dm4t2.github.io/vue-currency-input/config.html#currency

**Reply by HarshikaSM** [reply]
The currency code is getting displayed instead of the currency symbol. This is the config and the value being displayed.
`
{"locale":"yo-NG",
"currency":"NGN",
"valueAsInteger":false,
"hideGroupingSymbol":false,
"hideCurrencySymbol":false,
"hideNegligibleDecimalDigits":true,
"precision":0,
"autoDecimalMode":false,
"allowNegative":true
}`

![Screenshot from 2023-11-27 11-42-08](https://github.com/dm4t2/vue-currency-input/assets/151844178/27c64c05-57a9-48f6-9154-804b48973d62)

Also created anpther example for reference: https://codepen.io/HarshikaSM/pen/wvNxgVK

**Reply by dm4t2** [reply]
For some reason, the currency symbol "₦" is only displayed with `{currencyDisplay: 'narrowSymbol'}`.
Please note, that problems with [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) should be reported at https://github.com/tc39/ecma402.

**Reply by HarshikaSM** [reply]
If i change the prefix in CurrencyInputOptions , then the symbol is getting displayed.
